tests/plugins/test_ddp_plugin.py

```diff
@@ -69,3 +69,29 @@ def test_ddp_barrier_non_consecutive_device_ids(barrier_mock, tmpdir):
     trainer = Trainer(default_root_dir=tmpdir, max_steps=1, gpus=gpus, accelerator="ddp")
     trainer.fit(model)
     barrier_mock.assert_any_call(device_ids=[gpus[trainer.local_rank]])
+
+
+class BoringModelDDP(BoringModel):
```

Comment:
This is a very nice class for verifying when the model is expected to be wrapped. nit: maybe move it to a common place so it can be dedup'd among the DPP plugin tests?

Reply:
good idea. I will adopt this idea in another PR https://github.com/PyTorchLightning/pytorch-lightning/pull/8632